File: tests/slicing/adpcm.ml
(*
ledit bin/toplevel.top  -no-annot -deps -slicing_level 2 tests/slicing/adpcm.c
#use "tests/slicing/select.ml";;
*)

include LibSelect;;

(* Kernel.slicing_level := 2;;  = MinimizeNbCalls *)

(*
let resname = "tests/slicing/adpcm.sliced" in
ignore (test "uppol2" ~do_prop_to_callers:true ~resname (select_retres));;
*)
let () =
  Db.Main.extend
    (fun _ -> ignore (test "uppol2" ~do_prop_to_callers:true (select_retres)))

File: tests/slicing/adpcm.c
/* run.config
   EXECNOW: make -s @PTEST_DIR@/@PTEST_NAME@.cmxs
   STDOPT: +"-load-module ./tests/slicing/libSelect.cmxs -load-module @PTEST_DIR@/@PTEST_NAME@.cmxs -ulevel -1 -deps -slicing-level 2 -journal-disable"
*/

#include "tests/test/adpcm.c"
